.github/workflows/build.yml

```
@@ -77,11 +77,23 @@ jobs:
         docker push ${{env.DOCKER_REPO}}:${{env.BRANCH_TAG}}
         docker push ${{env.DOCKER_REPO}}:${{env.SHA_TAG}}
 
-
     - name: Trigger QA Deployment
       if: ${{ success() && github.ref == 'refs/heads/master' }}
       uses: benc-uk/workflow-dispatch@v1.1
       with:
         workflow: 'Deploy to PaaS'
         token: ${{ secrets.API_TOKEN_FOR_GITHUB_ACTION }}
         inputs: '{"environment": "qa", "sha": "${{ env.SHA_TAG }}"}'
+
+
+    - name: Check for Failure
+      if: ${{ always() && github.ref == 'refs/heads/master' }}
```

```suggestion
      if: ${{ failure() && github.ref == 'refs/heads/master' }}
```
also pls remove the extra line break 87/88